checkov/terraform/runner.py

```diff
@@ -120,24 +121,29 @@ def get_graph_checks_report(self, root_folder, breadcrumbs, runner_filter: Runne
             registry_results = r.run_checks(self.graph_manager.get_reader_traversal(), runner_filter)
             checks_results = {**checks_results, **registry_results}
 
-        for check_id, check_results in checks_results.items():
+        for check, check_results in checks_results.items():
             for check_result in check_results:
                 entity = check_result['entity']
                 entity_context, entity_evaluations = self.get_entity_context_and_evaluations(entity)
                 if entity_context:
                     full_file_path = entity[CustomAttributes.FILE_PATH]
                     copy_of_check_result = copy.deepcopy(check_result)
+                    for skipped_check in entity_context['skipped_checks']:
```

Comment:
@nimrodkor can you please add a UT for it?

Reply:
Added